sumdbaudit/witness/cmd/feeder/main.go

```diff
@@ -66,8 +66,13 @@ func main() {
 		}
 	}
 
+	lid := *logID
```

Comment:
super nit: probably should be `lID` (although it's harder to read :/)

Reply:
The Go gods might be cross with me, but I'll stick with `lid` - it parses so much easier.